sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -322,3 +322,47 @@ SELECT INTERVAL '153722867280' MINUTE;
 SELECT INTERVAL '-153722867280' MINUTE;
 SELECT INTERVAL '54.775807' SECOND;
 SELECT INTERVAL '-54.775807' SECOND;
+
+SELECT INTERVAL '1' DAY > INTERVAL '1' HOUR;
```

Comment:
I'd only keep
```
SELECT INTERVAL '1' DAY > INTERVAL '1' HOUR;
SELECT INTERVAL '1 02' DAY TO HOUR = INTERVAL '02:10:55' HOUR TO SECOND;
SELECT INTERVAL '1' YEAR < INTERVAL '1' MONTH;
SELECT INTERVAL '-1-1' YEAR TO MONTH = INTERVAL '-13' MONTH;

SELECT array(INTERVAL '1' YEAR, INTERVAL '1' MONTH);
SELECT array(INTERVAL '1' DAY, '01:01' HOUR TO MINUTE);
SELECT coalesce(INTERVAL '1' YEAR, INTERVAL '1' MONTH);
SELECT coalesce(INTERVAL '1' DAY, '01:01' HOUR TO MINUTE); 
```

Reply:
You can test more combinations in `TypeCoercionSuite`

Reply:
> You can test more combinations in `TypeCoercionSuite`

Done